GitExtensionsVSIX/source.extension.vsixmanifest

```diff
@@ -24,4 +26,7 @@
     <Assets>
         <Asset Type="Microsoft.VisualStudio.VsPackage" d:Source="Project" d:ProjectName="%CurrentProject%" Path="|%CurrentProject%;PkgdefProjectOutputGroup|" />
     </Assets>
+    <Prerequisites>
+        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[14.0,16.0)" DisplayName="Visual Studio core editor" />
```

Comment:
Not sure the impact, but just to let you know this looks different the the official [migration guide](https://docs.microsoft.com/en-gb/visualstudio/extensibility/how-to-migrate-extensibility-projects-to-visual-studio-2017?utm_source=hs_email&utm_medium=email&utm_content=39368033&_hsenc=p2ANqtz-9L_AqZ6evgRtUkVe9UUsBkjGZwNYjXGNs--mH-3Top6yJna5IlDY4Zvuy4-vEsLs0CELuA6Js6J7kWHEBjAz84UlM0Rg&_hsmi=39368033), which has two more prerequisites and the version of the min version of CoreEditor is 15.0.

```
    <Prerequisites>
        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,16.0)" />
        <Prerequisite Id="Microsoft.VisualStudio.Component.DiagnosticTools" Version="[15.0.25814.0,16.0)" />
        <Prerequisite Id="Microsoft.VisualStudio.Shell.12.0" Version="[12.0]" />
    </Prerequisites>
```
```